docs/routers/openwrt.md

```diff
@@ -0,0 +1,76 @@
+# OpenWrt
+
+Here's a short guide to configure Pi-hole as the DNS server for your OpenWrt router.
+This guide was written for OpenWrt version 24.10 but the principle should hold for any earlier or later release although Web UI might look different.
+
+## Requirements
```

Comment:
I think this whole section is not needed. It should be clear that a guide for OpenWrt & Pi-hole requires an OpenWrt router and Pi-hole....